packages/editor/src/browser/semantic-highlight/semantic-highlighting-service.ts

```diff
@@ -44,29 +44,25 @@ export class SemanticHighlightingService implements Disposable {
      */
     register(languageId: string, scopes: string[][] | undefined): Disposable {
         if (scopes && scopes.length > 0) {
-            this.logger.info(`Registering scopes for language: ${languageId}.`);
+            this.logger.debug(`Registering scopes for language: ${languageId}.`);
```

Comment:
I believ this information is as relevant as any `Started watching` (`nsfw`) events, please change it back from `debug` to `info`. What do you think?

Reply:
I will revert it. I think that logging watching of each opened file is super loud, but it is out of scope.